builtin/v9/miner/miner_state.go

```diff
@@ -178,6 +189,20 @@ func (st *State) GetInfo(store adt.Store) (*MinerInfo, error) {
 	return &info, nil
 }
 
+type BeneficiaryTerm struct {
+	Quota      abi.TokenAmount
+	UsedQuota  abi.TokenAmount
+	Expiration abi.ChainEpoch
+}
+
+type BeneficiaryChange struct {
```

Comment:
I'd either match the name in builtin actors, or PR a change to the name there (I think this name is better tbh).